src/test/developTest/resources/wdr/wdr_sendung_overview_lokalzeit.html

```diff
@@ -0,0 +1,1472 @@
+</head>
+<body class="wdr themen viewL" >
+<div id="header" >
+<div class="wrapper">
+<div class="section sectionA" data-ctrl-layoutable="{'id':'laySub','action':{'default':['widthBound'],'s':['disable'],'xs':['disable']}}" data-ctrl-collapsible="{'id':'collHead','action':{'default':['openEntries','disable'],'s':['closeEntries','closeEntriesOnVanish','enable'], 'xs':['closeEntries','closeEntriesOnVanish','enable']}}">
+<div data-ctrl-layoutable="{'id':'laySub','action':{'default':['widthBound'],'s':['disable'],'xs':['disable']}}" class="mnHolder">
+<div class="fixedHeader">
+<div class="mnHolder">
+<a class="hidden" name="goToHead"></a>
+<ul id="skipLinks">
+<li>
+<a class="hidden" href="#goToContent">zum Inhalt</a>
+</li>
+<li>
+<a class="hidden" href="#wdrNavi">zur Navigation</a>
+</li>
+<li>
+<a class="hidden" href="#wdrSuche">zur Suche</a>
+</li>
+</ul>
+<div role="banner" class="logo">
+<a href="/index.html" >
+<img alt="zu Startseite WDR" src="/resources/img/wdr/logo/wdr_logo.svg" title="zu Startseite WDR"/>
+</a>
+</div>
+<div class="servicenavi" role="navigation">
+<ul data-ctrl-collhead-h3-body="{}">
+<li class="wetter">
+<a href="/wetter/index.html" title="Startseite Wetter">
+<span>Wetter</span>
+</a>
+</li>
+<li class="verkehr">
+<a href="/verkehr/index.html" title="Startseite Verkehr">
+<span>Verkehr</span>
+</a>
+</li>
+</ul>
+</div>
+<div role="search" class="searchBox" data-ctrl-collHead-entry="{'id':'ab4f1660-3f7b-4072-820d-b198e24cbab8-id'}">
+<a id="wdrSuche" class="hidden"></a>
+<div class="searchToggle" data-ctrl-collHead-ab4f1660-3f7b-4072-820d-b198e24cbab8-id-trigger="{'action':{'s':['closeOtherEntries','toggleBody'], 'xs':['closeOtherEntries','toggleBody']}}">
+<a href="/suche/index.jsp">Suche</a>
+</div>
+<form name="form_ab4f1660-3f7b-4072-820d-b198e24cbab8-id"
+class="searchForm"
+method="post"
+action="/suche/index.jsp"
+data-ctrl-collHead-ab4f1660-3f7b-4072-820d-b198e24cbab8-id-body={}
+data-ctrl-formular="{'id':'form_ab4f1660-3f7b-4072-820d-b198e24cbab8-id'}">
+<input
+class="searchInput"
+name="q"
+type="text"
+value="im WDR suchen"
+data-ctrl-form_ab4f1660-3f7b-4072-820d-b198e24cbab8-id-input="{'action':{'default':['switchValue']}}"
+/>
+<a	href="/suche/index.jsp"
+class="searchSubmit"
+data-ctrl-form_ab4f1660-3f7b-4072-820d-b198e24cbab8-id-trigger="{'action':{'default':['submit']}}">
+</a>
+</form>
+</div>
+<div class="naviToggle menueToggle" data-ctrl-collhead-h0-trigger="{'action':{'s':['closeOtherNavis','toggleBody'], 'xs':['closeOtherNavis','toggleBody']}}">
+<a href="#">Menü</a>
+</div>
+</div>
+</div>
+<div class="naviCont" data-ctrl-collhead-entry="{'id':'h0'}">
+<div class="navis" data-ctrl-collhead-h0-body="{}">
+<div role="navigation" class="subnavi" data-ctrl-navigation="{'id':'csub','action':{'default':['closeEntries','closeEntriesOnVanish']}}">
+<a class="hidden" name="wdrNavi" id="wdrNavi"></a>
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this deprecated "name" attribute. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=Web%3AS1827)